### DISCUSSION TITLE
remoteEnty.js file not found in development mode

### DISCUSSION THREAD

**Discussion by stephencranedesign** [originjs/vite-plugin-federation#455] [Q&A] [answered]
I have a really basic vite project from running `npm create vite@latest`. After that I installed `@originjs/vite-plugin-federation`.

My vite config looks like this:

````js
import { defineConfig } from 'vite';
// import react from '@vitejs/plugin-react-swc';
import federation from '@originjs/vite-plugin-federation';

// https://vitejs.dev/config/
export default defineConfig({
    mode: 'development',
  plugins: [
      // react(),
      federation({
          name: 'remote1',
          filename: 'remoteEntry.js',
          exposes: {
              './App': './src/App',
          },
          shared: [],
      }),
  ],
});
````

The dev server starts fine on port 5173 but I can't find where the remoteEntry.js file is. I've tried: `http://localhost:5173/remoteEntry.js` and `http://localhost:5173/assets/remoteEntry.js`.

Where's it suppose to be and how do I get it to be created?

**Comment by sohaibaumer** [comment, marked as answer]
npm run build

'http://localhost:5173/dist/remoteEntry.js'

**Reply by mahdihssni** [reply]
Should I build the remote app for each change?

**Reply by mahdihssni** [reply]
Could I change the directory of output?
